src/ToricVarieties/AttributesAndMethods.jl

```diff
@@ -138,3 +500,98 @@ export ith_betti_number
 #     return result
 # end
 # export toric_ideal_binomial_generators
+
+
+############################
+# Advanced constructions
+############################
+
+@doc Markdown.doc"""
+    blowup_on_ith_minimal_torus_orbit(v::AbstractNormalToricVariety, n::Int)
+
+Computes the blowup of the normal toric variety `v` on its i-th minimal torus orbit.
+
+# Examples
+```jldoctest
+julia> P2 = toric_projective_space(2)
+A normal toric variety corresponding to a polyhedral fan in ambient dimension 2
+
+julia> blowup_on_ith_minimal_torus_orbit(P2,1)
+A normal toric variety corresponding to a polyhedral fan in ambient dimension 2
+```
+"""
+function blowup_on_ith_minimal_torus_orbit(v::AbstractNormalToricVariety, n::Int)
+    return NormalToricVariety( starsubdivision( fan_of_variety( v ), n ) )
+end
+export blowup_on_ith_minimal_torus_orbit
+
+
+@doc Markdown.doc"""
+    Base.:*(v::AbstractNormalToricVariety, w::AbstractNormalToricVariety)
+
+Computes the Cartesian/direct product of two normal toric varieties `v` and `w`.
+
+# Examples
+```jldoctest
+julia> P2 = toric_projective_space(2)
+A normal toric variety corresponding to a polyhedral fan in ambient dimension 2
+
+julia> P2 * P2
+A normal toric variety corresponding to a polyhedral fan in ambient dimension 4
+```
+"""
+function Base.:*(v::AbstractNormalToricVariety, w::AbstractNormalToricVariety)
+    return NormalToricVariety(fan_of_variety(v)*fan_of_variety(w))
+end
+export *
+
+
+############################
+# Comparison
+############################
+
+
+@doc Markdown.doc"""
+    isprojective_space(v::AbstractNormalToricVariety)
+
+Decides if the normal toric varieties `v` is a projective space.
+
+# Examples
+```jldoctest
+julia> H5 = hirzebruch_surface(5)
+A normal toric variety corresponding to a polyhedral fan in ambient dimension 2
+
+julia> isprojective_space(H5)
+false
+
+julia> isprojective_space(toric_projective_space(2))
+true
+```
+"""
+function isprojective_space(v::AbstractNormalToricVariety)
+    if issmooth(v) == false
+        return false
+    end
+    if isprojective(v) == false
+        return false
+    end
+    if rank(class_group(v)) > 1
+        return false
+    end
+    w = [[Int(x) for x in transpose(g.coeff)] for g in gens(class_group(v))]
+    for i in 1:length(w)
+        g = w[i]
+        g = g[findall(x -> x!=0, g)]
```

Comment:
Something like this may also work:
```suggestion
    for g in gens(class_group(v))
        g = [Int(x) for x in transpose(g.coeff) if !iszero(x)]
```
Actually, do you really need the call `transpose`?

Reply:
This is a good suggestion. Also, I indeed think the transpose can be dropped.

Reply:
Done in https://github.com/oscar-system/Oscar.jl/pull/755/commits/c6c9ab874dee31b54b37400769f98bc660ec0a0e.